src/main/java/servlets/CreateServlet.java

```diff
@@ -0,0 +1,98 @@
+// limitations under the License.
+package servlets;
+
+import data.JsonConverter;
+import data.Vendor;
+import data.Account;
+import data.SheetsConverter;
+import static servlets.VendorServlet.updateSheets;
+
+import com.google.gson.Gson;
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.security.GeneralSecurityException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+
+@WebServlet("/CreateServlet")
+public class CreateServlet extends HttpServlet {
+  public static final String CONTENT_TYPE_APPLICATION_JSON = "application/json;";
+  public static final String REDIRECT_EDITFILE = "/editfile.html";
+  private static final String TOKEN_ATTRIBUTE = "accessToken";
+  private static final String ADD_ACCOUNT_REDIRECT = "/add-account.html";
+  private final String VENDOR_ID_PARAM = "vendorID";
+  private final String LEGACY_CUSTOMER_ID_PARAM = "legacycustomerID";
+  private final String NEXT_GEN_CUSTOMER_ID_PARAM = "nextgencustomerID";
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // POST method --> creates a new configuration file with a pre-made json config 
+    // with the given vendor id and account id and all other values as null.
+    JsonConverter converter = new JsonConverter();
+    Vendor vendor = new Vendor();
+    String vendorID = request.getParameter(VENDOR_ID_PARAM);
+    System.out.println(VENDOR_ID_PARAM);
+    String legacyCustomerID = request.getParameter(LEGACY_CUSTOMER_ID_PARAM);
+    String nextGenCustomerID = request.getParameter(NEXT_GEN_CUSTOMER_ID_PARAM);
+    String responseString = String.format(
+      "New Config with VendorID: %s was successfully created.", vendorID);
+
+    // Set initial ids on the new vendor
+    int nextGenCustomerIDInt = Integer.parseInt(nextGenCustomerID);
+    vendor.setVendorID(vendorID);
+    vendor.setLegacyVendorID(legacyCustomerID);
+    vendor.setNextGenVendorID(nextGenCustomerIDInt);
+
+    // Check if file doesn't already exist within the file system.
+    try {
+      ArrayList currentVendors = converter.getVendorIDs();
+      if (currentVendors.contains(vendorID) == true) {
+        System.out.println("File is here");
+        String jsonMessage = new Gson().toJson("File already exists in file system.");
+        response.setContentType(CONTENT_TYPE_APPLICATION_JSON);
+        response.getWriter().println(jsonMessage);
+      } else {
+        String jsonConfig = vendor.buildJsonConfig();
+        
+        // Creates the new vendor configuration file.
+        File newFile = converter.writeFile(vendorID, jsonConfig);
```

Comment:
Can you make a method similar to [updateFile()](https://github.com/googleinterns/step57-2020/blob/85d660789e3ceca7256f4cef26e9dbc69dc2343d/src/main/java/data/JsonConverter.java#L37-L53) that checks to see if our configuration exists before we write to it. We don't want to allow the user to overwrite a file that already exists when they create, and that could allow us to do some error handling